### PR TITLE
Tweak default front-matter.

### PR DESCRIPTION
**Pull request by therealfakemoot**
I noticed new posts weren't having their date/author being populated, so I just threw in a few variables into the archetype.

Regarding `+++` versus `---` I don't *actually* know if the former is invalid or not, but I'm used to seeing `---` so I changed that to be safe.

Here's some example front-matter

```
$ cat content/posts/times-arrow.md
---
title: "Times Arrow"
date: "2020-03-13T09:46:32-04:00"
draft: true
author: 'Nick Dumas'
cover: ""
tags: ["", ""]
keywords: ["", ""]
description: ""
showFullContent : false
---
```

### CODE REVIEW

**Comment by panr** [comment]
I use `TOML` in my theme and you unfortunately want to change a front matter syntax to `YAML` ;-(

But you're right that we should populate some of the params with values from the `config` file. But I don't remember if this works like that. Can you confirm that the archetype can read the `config` file?

**Comment by therealfakemoot** [comment]
My apologies, I don't know how I missed the frontmatter being TOML. My latest commit has changed that back. `hugo build` runs without issues so I believe it's good.

File: archetypes/posts.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 +++
-title = ""
-date = ""
-author = ""
-authorTwitter = "" #do not include @
+title = "{{ replace .Name "-" " " | title }}"
+date = "{{ .Date }}"
+draft = true
+author = '{{ index .Site.Author "name" }}'
```
Comment (panr) [inline archetypes/posts.md:5]: Is it okay to wrap this in a single quote?

Comment (therealfakemoot) [reply archetypes/posts.md:5]: I did a test with single quotes and it seems to generate just fine. I've changed all string literals to be enclosed by single quotes.

**Comment by panr** [comment]
Oh, why did you remove `authorTwitter = "" #do not include @`?

**Comment by panr** [comment]
I'm closing the PR due to the lack of a response 😢

**Pull request closed without merging**